### PR TITLE
Add sign up related views.

### PR DESCRIPTION
**Pull request by sylmak**
Add sign up related views. Tests to come in the future.

### CODE REVIEW

File: src/components/CommonDisclaimer.vue [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<template>
+  <p>
+    By continuing, you acknowledge that you understand and agree to the
+    <!-- TODO: Update links -->
+    <RouterLink to="/">Terms & Conditions</RouterLink> and
+    <RouterLink to="/">Privacy Policy</RouterLink>.
```
Comment (skeshive) [inline src/components/CommonDisclaimer.vue:6]: fyi the privacy policy is ready, feel free to add it in a follow up CL

File: src/components/SignUp.vue [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  <form @submit.prevent>
+    <div class="email">
+      <label for="email">Email</label>
+      <input type="text" name="email" v-model="email" />
+      <p>We'll send you an email verification link.</p>
+    </div>
+
+    <div class="password">
+      <label for="password">Password</label>
+      <input type="password" name="password" v-model="password" />
```
Comment (skeshive) [inline src/components/SignUp.vue:46]: are there any Firebase restrictions for the password? if so, should we enforce the same restrictions?

Comment (sylmak) [reply src/components/SignUp.vue:46]: There are no restrictions currently. We can set up restrictions if we enable Identity Platform. Pinged you the chat thread and will add a task in Notion for this.

File: src/components/SignUpCheckEmail.vue [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  }
+});
+
+onBeforeUnmount(unsubscribeAuthListener);
+</script>
+
+<template>
+  <h2>Check your email</h2>
+  <p>
+    We've sent an email to {{ email }}. Please click on the link in your email
```
Comment (skeshive) [inline src/components/SignUpCheckEmail.vue:34]: nit: this text doesnt match mocks ("We've sent you an email to x.")

File: src/components/SignUpCheckEmail.vue [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    to verify.
+  </p>
+  <p>
+    Didn't receive the mail? Check your spam filter for an email from
```
Comment (skeshive) [inline src/components/SignUpCheckEmail.vue]: nit: email

File: src/components/SignUpCheckEmail.vue [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    account@climateiq.com.
+  </p>
+  <button @click="resendEmail">Resend email</button>
+  <div class="account-actions">
+    <p><RouterLink to="/sign-up">Use another email</RouterLink></p>
+    <p><RouterLink to="/login">Log in</RouterLink></p>
+  </div>
+
+  <div class="need-help">
```
Comment (skeshive) [inline src/components/SignUpCheckEmail.vue]: this section isn't in the mocks, was it discussed offline?

**Review by skeshive — APPROVED** [review]

File: src/components/SignUpCheckEmail.vue (hunk 3 above)
Comment (sylmak) [reply src/components/SignUpCheckEmail.vue:34]: It doesn't sound grammatically correct to me - I think it's either "We've sent an email to {{ email }}" or "We've sent you an email at {{ email }}".

File: src/components/SignUpCheckEmail.vue (hunk 5 above)
Comment (sylmak) [reply src/components/SignUpCheckEmail.vue]: Good catch, removed

**Pull request merged**